src/redux/configureStore.js

```diff
@@ -11,7 +12,7 @@ const reducer = combineReducers({
 
 const store = createStore(
   reducer,
-  applyMiddleware(logger),
+  applyMiddleware(logger, thunk),
```

Comment:
- [ ] Please make sure to meet the following requirement 💪🏽

- [ ] All Bookstore API calls are done with fetch OR Axios.
Action Creator for fetching books should be created

- [ ] It should return a function.

- [ ] It should fetch the books list from API.

- [ ] It should dispatch a new action type that will pass the books list to reducer and updates state.


- [ ] Action creator for adding a book should be refactored:

- [ ] It should return a function.

- [ ] It should add a new book via API.

- [ ] It should dispatch already existing ADD_BOOK (name can be slightly different).


- [ ] Action creator for removing a book should be refactored:

- [ ] It should return a function.

- [ ] It should remove a book via API.

- [ ] It should dispatch already existing REMOVE_BOOK (name can be slightly different).


- [ ] The Add book and remove book features should work the same as before even after the refactoring to use the API has been done.